shared/src/business/utilities/getCaseDocumentsIdsFilteredByDocumentType.ts

```diff
@@ -0,0 +1,29 @@
+export const getCaseDocumentsIdsFilteredByDocumentType = (
+  applicationContext,
+  {
+    docketEntries,
+    docketRecordFilter,
+    documentsToProcess,
+  }: {
+    docketEntries: RawDocketEntry[];
+    docketRecordFilter: string;
+    documentsToProcess: { docketEntryId: string }[];
+  },
+): string[] => {
+  const formattedDocketEntries = documentsToProcess.map(docSelected => {
+    return docketEntries.find(
+      docEntry => docEntry.docketEntryId === docSelected.docketEntryId,
+    );
+  });
```

Comment:
I'm having a hard time understanding the utility of this block and function. Can you help me understand it?

What's the difference between documentsToProcess and docketEntries?